demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -180,7 +181,19 @@ class BaseContent(BaseNode):
     git_status: Optional[GitStatuses]
     old_base_content_object: Optional["BaseContent"] = None
 
-    def _save(self, path: Path, data: dict):
+    def _save(
+        self,
+        path: Path,
+        data: dict,
+        predefined_keys_to_keep: Optional[Tuple[str]] = None,
```

Comment:
```suggestion
        predefined_keys_to_keep: Optional[Tuple[str], ...] = None,
```
the type-hint here means we can only get tuple of one value
this way its a tuple of any size
please check if it works